### PR TITLE
Add a value range

### PR DESCRIPTION
**Pull request by Bouh**
I was like "What should I type here"?

### CODE REVIEW

File: Extensions/TweenBehavior/JsExtension.js [hunk 1]
```diff
@@ -855,9 +855,9 @@ module.exports = {
       .addParameter('expression', _('To Hue'), '', false)
       .addParameter('yesorno', _('Animate Hue'), '', false)
       .setDefaultValue('yes')
-      .addParameter('expression', _('To Saturation (-1 to ignore)'), '', false)
+      .addParameter('expression', _('To Saturation (-1 to ignore to 100)'), '', false)
       .setDefaultValue('-1')
-      .addParameter('expression', _('To Lightness (-1 to ignore)'), '', false)
+      .addParameter('expression', _('To Lightness (-1 to ignore to 100)'), '', false)
```
Comment (AlexandreSi) [inline Extensions/TweenBehavior/JsExtension.js:860]: What do you think about `(0 to 100, -1 to ignore)`?

Comment (Bouh) [reply Extensions/TweenBehavior/JsExtension.js:860]: I agree 👍

File: Extensions/TweenBehavior/JsExtension.js [hunk 2]
```diff
@@ -855,9 +855,9 @@ module.exports = {
       .addParameter('expression', _('To Hue'), '', false)
       .addParameter('yesorno', _('Animate Hue'), '', false)
       .setDefaultValue('yes')
-      .addParameter('expression', _('To Saturation (-1 to ignore)'), '', false)
+      .addParameter('expression', _('To Saturation (-1 to ignore to 100)'), '', false)
```
Comment (Bouh) [inline Extensions/TweenBehavior/JsExtension.js:858]:
```suggestion
      .addParameter('expression', _('To Saturation (0 to 100, -1 to ignore)'), '', false)
```

File: Extensions/TweenBehavior/JsExtension.js (hunk 1 above)
Comment (Bouh) [inline Extensions/TweenBehavior/JsExtension.js:860]:
```suggestion
      .addParameter('expression', _('To Lightness (0 to 100, -1 to ignore)'), '', false)
```

**Review by AlexandreSi — APPROVED** [review]
LGTM

**Pull request merged**